test/test-filter-process.sh

```diff
@@ -6,15 +6,7 @@
 # from the "next" branch, which is the only (current) version of Git that has
 # support for the filter protocol.
 #
-# Once 2.11 is released, replace this with:
-#
-# ```
-# ensure_git_version_isnt $VERSION_LOWER "2.11.0"
-# ```
-if [ "1" -ne "$(git version | cut -d ' ' -f 3 | grep -c "g")" ]; then
-  echo "skip: $0 git version does not include support for filter protocol"
-  exit
-fi
+ensure_git_version_isnt $VERSION_LOWER "2.11.0"
```

Comment:
These "custom hook paths" tests have two parts:

- The `test/test-install-custom-hooks-path-unsupported.sh` runs in versions of Git that _don't_ support custom hooks paths (i.e., Git <2.9.0), and ensures that the custom hooks directory is ignored, and the hooks are placed in `.git/hooks/`.
- the `test/test-install-custom-hooks-path.sh` runs on versions of Git that _do_ support custom hooks paths (i.e., Git >=2.9.0), and ensures that the custom hooks directory is respected. 

There's no (current) clean way to do this in one file, so I split it into two, and only one of them runs at a time.